test/regression/tests_utils.py

```diff
@@ -70,48 +71,3 @@ def get_default_copy_cmd(db_path, filename, experiment_id):
 
 def get_conf_file_path(base_path, filename):
     return os.path.join(base_path, 'conf', filename + '.conf')
-
-
-def base36encode(number, alphabet=string.digits + string.ascii_lowercase):
-    """
-    Convert positive integer to a base36 string.
-
-    :param number: number to convert
-    :type number: int
-    :param alphabet: set of characters to use
-    :type alphabet: str
-    :return: number's base36 string value
-    :rtype: str
-    """
-    if not isinstance(number, int):
-        raise TypeError('number must be an integer')
-
-    # Special case for zero
-    if number == 0:
-        return '0'
-
-    base36 = ''
-
-    sign = ''
-    if number < 0:
-        sign = '-'
-        number = - number
-
-    while number > 0:
-        number, i = divmod(number, len(alphabet))
-        # noinspection PyAugmentAssignment
-        base36 = alphabet[i] + base36
-
-    return sign + base36.rjust(4, '0')
-
-
-def base36decode(number):
-    """
-    Converts a base36 string to a positive integer
-
-    :param number: base36 string to convert
-    :type number: str
-    :return: number's integer value
-    :rtype: int
-    """
-    return int(number, 36)
```

Comment:
Duplicate from `experiment_common`. Deleted the copy in the test file, used the real code to test (not sure why it was done the other way).